### PR TITLE
added basic join_skip page

### LINKED ISSUE #4: Add 'Join SKIP' subpage
## Add 'Join SKIP' subpage

Add the _'Join SKIP'_ subpage that enables students to register as `SKIP` members and/or board members. Follow the instructions shared in the document.

### PR DESCRIPTION
**Pull request by JonathanPartain**
Feedback and/or improvements welcome, just a basic page. Link is missing, currently WIP according to document. Aims to solve #4

### CODE REVIEW

File: join_skip.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+1. **Regular Members:**
+As a regular member, you can actively contribute to the success of SKIP. Your participation in events and feedback are highly valued. Additionally, you'll enjoy perks like priority access to events and discounts. If you're ready to dive in and be an essential part of the SKIP experience, apply today by clicking
+
+2. **Board Members:**
+If you're passionate about event planning, leadership, and making a significant impact, consider joining our board. Board members have a crucial role in shaping the direction of SKIP. Please note that board member applications are individually reviewed. If selected, you will be scheduled for an onboarding process to help you transition into your leadership role effectively. To apply for a board position, click [LINK](link_goes_here)!
+
+Should you have any questions or require further information, please don't hesitate to reach out to us at skip@gota.gu.se. We look forward to welcoming you as part of the SKIP family!
```
Comment (michalspano) [inline join_skip.md]: I'd **suggest** to use: [*skip@gota.gu.se*](mailto:skip@gota.gu.se).

File: join_skip.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+title: Join SKIP
+layout: post
+---
+
+## Join Us: Become a SKIP Member!
+
+Are you eager to be part of a dynamic and impactful student association? At SKIP, we welcome students who are enthusiastic about enhancing the academic and social experience of their peers. Whether you're interested in taking a leadership role as a board member or providing valuable support as a regular member, there's a place for you in our community.
+
+1. **Regular Members:**
+As a regular member, you can actively contribute to the success of SKIP. Your participation in events and feedback are highly valued. Additionally, you'll enjoy perks like priority access to events and discounts. If you're ready to dive in and be an essential part of the SKIP experience, apply today by clicking
```
Comment (michalspano) [inline join_skip.md]: Looks good! Just the link is missing.

**Review by michalspano — CHANGES_REQUESTED** [review]
Left two small comments! Good job.

File: join_skip.md (hunk 1 above)
Comment (JonathanPartain) [reply join_skip.md]: Are you referring to the email being a mailto: link, or do you want it in italics?

File: join_skip.md (hunk 2 above)
Comment (JonathanPartain) [reply join_skip.md]: Wanna leave this PR open until a link exists?

File: join_skip.md (hunk 1 above)
Comment (michalspano) [reply join_skip.md]: Oh, both, without the rendering:
```md
[*skip@gota.gu.se*](mailto:skip@gota.gu.se)
```

Comment (michalspano) [reply join_skip.md]: I see it's correctly implemented now!

**Review by michalspano — APPROVED** [review]
Good job! This is ready to be **merged**.

**Pull request merged**